src/ripple/app/tests/MultiSign.test.cpp

```diff
@@ -621,6 +621,39 @@ class MultiSign_test : public beast::unit_test::suite
         env(regkey (alice, disabled), sig(alie));
     }
 
+    // Verify that the first regular key can be made for free using the
+    // master key, but not when multisigning.
+    void test_regKey()
+    {
+        using namespace jtx;
+        Env env(*this);
+        Account const alice {"alice", KeyType::secp256k1};
+        env.fund(XRP(1000), alice);
+
+        // Give alice a regular key with a zero fee.  Should succeed.  Once.
+        Account const alie {"alie", KeyType::ed25519};
```

Comment:
Tests generally don't need to specify they keytype unless you are testing something specific to that key type. It doesn't hurt, but it doesn't really add anything to the test either.